Add blog post for June Standard community call (#351)
publiccodenet/blog: _posts/2022-06-08-notes-from-community-call-2-june-2022.md

This may be expressed differently depending on the role of the person searching.

Besides a unique name and basic discoverability, an at least minimal web page would be a good requirement for any codebase that has the ambition to meet the Standard for Public Code and be reused.

Beyond that, it was unclear if any further MUSTs or SHOULDs are needed.
MAY requirements could be more encouraging than just guidance since that would give civil servants more justification to do it and also a reward in terms of an additional ticked box.

Many other aspects were discussed, like contact persons, use of metadata standards, software catalogs, social media and domain names.
Most of these could be MAY requirements, though we didn't have sharp suggestions of phrasing during the call, whereas others should probably not be requirements at all.
For example, we think that social media may be a suboptimal strategy for communication for a public code codebase.

> The similarities to the development of open data was also noted

Reviewer: ```suggestion
The similarities to the development of open data were also noted
```

It has been shown that only creating good metadata was not enough.

> For it to really be reused, more proactivity is needed and we should be inspired by from the experiences in that movement.

Reviewer: ```suggestion
For it to really be reused, more proactivity is needed and we should be inspired by the open data movements' experiences.
```
